setup.py

```diff
@@ -23,6 +23,8 @@ def read(fname):
 install_requires += (["requests"],)
 install_requires += (["scikit-image"],)
 install_requires += (["toolz"],)
+install_requires += (["jsonschema"],)
+install_requires += (["ngff"],)
```

Comment:
Same as above, I think our project should be prefixed with `ome` i.e.` ome-ngff` inline with the differentiation between NGFFs and OME-NGFF that we settled on in the paper